# Human seasonal coronaviruses, immunity, and antigenic drift

- [Hamre1966, A new virus isolated from the human respiratory tract.](https://journals.sagepub.com/doi/10.3181/00379727-121-30734):
  First isolation of 229E.
  Isolated virus from five University of Chicago medical students, four with minor colds and one with no cold.
  They all yielded virus after a passage in human kidney cells.
  Virus 229E underwent thre serial selections at limiting dilution in WI-38 cells.
  Guinea pig sera to the virus neutralized all strains at about 1:1200.
  There were antibody rises of >4-fold in the sera of all the students from whom the viruses were isolated.

- [Bradburne1967, Effects of a "new" human respiratory virus in volunteers.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1843247/):
  Infection of human volunteers with 229E.
  First surveyed sero-prevalence of 229E neutralizing antibodies in Brits, and found that about 20-25% of children and 40% of adults had them--including 25% of children <=5 years.
  Also gives the passage history of 229E.
  Infected 26 volunteers, 13 of whom developed colds.
  Virus could be isolated from 13/13 who developed colds, but only 5/13 who were asymptomatic.
  Notably, the duration of shedding was no different between symptomatic and asymptomatic.
  Individuals with lower pre-existing neut antibody titers developed larger rises, and there appears to be a moderate trend towards those with higher pre-existing antibody developing fewer colds.

- [McIntosh1967](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC223830/pdf/pnas00152-0140.pdf):
  Reports isolation of viruses that look like IBV (avian infectious bronchitis virus) from patients with colds.
  Two of these patients, OC38 and OC43, had viruses that were grown in cell culture and passaged in mice brains.
  They say: "Serologic studies in organ culture suggested that the patients from whom the viruses were recovered developed a neutralizing antibody response, but further study of these viruses was impeded by the lack of a tissue culture or experimental animal system that would support their growth."
  They report how the viruses look similarly club shaped to IBV.
  They show that humans from whom viruses were collected have high neutralizing titers to both OC43 / OC38 and 229E.
  But mice infected with OC43 only produce neutralizing antibody responses to OC43, and not 229E.
  The titers are high: around 1:2000 in mice and at least 1:64 in humans.
  They also report seroprevalence studies saying that many patients with colds had these viruses, and that many of them had rises in complement-fixing antibody titers.

- [McIntosh1967b](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC224637/pdf/pnas00115-0087.pdf):
  Discusses isolation of IBV-like viruses (what we would now call OC43) from patients with colds.
  Convalescent sera neutralized OC43 at 1:4 (not sure if they tested less) but not 229E.
  Unlike 229E, they could only culture virus in human respiratory tract cultures, not cell culture.

  - [Hendley1972, Coronavirus infections in working adults. Eight-year study with 229E and OC43.](https://www.ncbi.nlm.nih.gov/pubmed/5020629):
    Followed insurance company employees over an 8-year period.
    Diagnosed infection if a >4-fold increase in antibody titer from acute (or often pre-infection) to convalescent (3-10 week) sera titers.
    229E serology was positive in 3% of colds from Dec to May, and 0.4% of colds in summer / autumn; during same time 1% of non-sick patients had titer increases.
    OC43 serology was positive for 5% of Dec to May colds and none in summer, and no antibody increases in non-sick.
    Winter (Dec - Feb) had similar infections as spring (March - May).
    Prevalence of antibody tended to increase with age, but were less than half (by their cutoff) even at 40+.
    Increases in titer to one of 229E or OC43 was not associated with increased titers to the other.
    Serologic evidence suggests some reinfections, although not sure how reliable this is?
    The symptoms of 229E seem more mild than those for OC43; 229E mostly limited to nasal symptoms.

- [Hamre1972, Virologic studies of acute respiratory disease in young adults. V. Coronavirus 229E infections during six years of surveillance.](https://www.ncbi.nlm.nih.gov/pubmed/4626012):
  Monitored University of Chicago medical students for infection with 229E-like viruses from 8 years.
  Infections were in winter-spring, and associated with respiratory illness.
  In three high-prevalence years between 15-35% of students had antibody rises; in other years between 1-5% did.
  There were always low prevalence years separating the high prevalence ones.
  Antibody rise was inversely correlated with pre-existing titers, but neutralizing antibody levels did not appear to affect likelihood of infection.
  Pre-infection neutralizing antibody titers were present at 1:2 to 1:15 pre-infection.
  After infection, most developed 4-fold rises with highest titer of 1:64.

- [McIntosh1974, Coronaviruses: A Comparative Review](https://link.springer.com/chapter/10.1007/978-3-642-65775-7_3):
  Very nice history of coronaviruses.
  IBV first found in chickens, then MHV in mice, then B814 and shortly thereafter 229E in humans.
  Makes a reference to well-known antigenic variation in IBV.
  Notes how OC43 agglutinates red blood cells.
  No observed antigenic variation in pig TGEV.
  IBV (chickens) appears to be antigenically quite variable.
  Some high level discussion of antigenic variation in human coronaviruses, but nothing concrete.

- [Monto1974, Medical reviews. Coronaviruses.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2595130/):
  Gives background: B814 isolated in UK and passaged in human organ culture and could still cause illness, then passaged in ferret culture and lost ability to cause illness.
  229E isolated in primary human kidney culture in 1962 from Chicago medical students.
  Points out that serology is complicated by difficulty isolating the virus.
  Study in Chicago found mean annual incidence of 229E was 15%.
  This varied a lot year to year, with 97% of infections between January and May.
  Study in Tecumseh found mean annual incidence of about 15% too.
  About 45% of infections estimated to give disease.
  Studies tend to find most adults have antibodies, but can still be re-infected.
  Children under 10 tend to have lower antibody titers.
  By third year of life, 50% of people have antibody against OC43.
  In general, big infection years are not consecutive with the same virus.
  Overall, suggests cycling every 2 to 4 years.
  For infections, mean time to symptoms is 3-4 days.
  Virus shedding usually begins with symptoms and lasts 1-4 days.
  In Tecumseh, 82% of those infected possessed prior neut antibody.

- [Kaye1975, Seroepidemiologic survey of coronavirus (strain 229E) infections in a population of children.](https://www.ncbi.nlm.nih.gov/pubmed/1115061):
  Studied 229E infections in a church-sponsored children's home using indirect hemagglutination test.
  1477 respiratory illnesses were reported over 8 years, of which 63 had 229E serological responses.
  There were also 105 serological responses to 229E without specific illnesses, although about 30% of these had some illness in the period.
  The highest fraction of the illnesses was in winter, intermediate in fall and spring, lowest in summer.
  Some years (always separated by low years) had higher peaks.
  14 children were in study for entire 8 years: in 1960 47% had antibody, peaked at 75% after high-incidence 1961-1962 season, back to 60^ by 1968.
  No trend for children with pre-existing antibody to have less illness--in fact, it's oppposite although difference probably not statistically significant.
  Also notes that 229E and OC43 have outbreaks in different years.
  They conclude: "The prevalence of preexisting antibody seemed to have no significant ameliorating effect on respiratory illness caused by 229E."

- [Isaacs1983, Epidemiology of coronavirus respiratory infections.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1628163/):
  Tested children with respiratory infections that presented to a pediatric department and their unaffected siblings.
  Overall, about 30% of the index children had coronavirus infection, mostly 229E, as did almost as many unaffected siblings.
  The index children tended to have lower respiratory tract infections, whereas the unaffected siblings all had only upper respiratory tract infections, often with some rhinitis.
  There were three peaks observed in the timeframe (1979 to 1981): two in Oct-Feb period, and one in May-Jul period.
  They also suggest re-infections are common, although they us an antigen test which also seems susceptible to just continued presence of antigen?

- [Reed1984, The Behaviour of Recent Isolates of Human Respiratory Coronavirus In Vitro and in Volunteers: Evidence of Heterogeneity Among 229E-Related Strains](https://onlinelibrary.wiley.com/doi/epdf/10.1002/jmv.1890130208):
  Compares 229E (isolated from 1962 specimen) with other related CoVs isolated from humans.
  Specifically: 229E - isolated 1962, LP - isolated 1965, KI - isolated 1974, PR - 1974 to 1976, TO - 1974 to 1976.
  Got sera from rabbits and mice immunized with 229E or LP.
  The neutralizing titers tended to be higher against the immunizing virus than the other strains, but the titer has only dropped about 1.5 to 3.5 log-2 units.
  It also appears that the other CoVs cause more cold symptoms than 229E, could this just be due to 229E being passed in culture more / longer?
  In human volunteers, infection with TO led to antibody rises against all antigens tested, but larger rises agains TO suggesting some specificity for that virus.
  They also re-inoculuated six subjects who had gotten colds and shed virus in first TO infection again with same TO virus after 8-12 months: on second infection none of them had colds or shed virus.
  But when they did similar experiments with a different virus, some of the subjects did get a cold and shed virus (5/8).

- [Callow1985, Effect of specific humoral immunity and some non-specific factors on resistance of volunteers to respiratory coronavirus infection.](https://www.ncbi.nlm.nih.gov/pubmed/2991366):
  Infected 33 volunteers with 229E.
  Both circulating and nasal antibodies protected from infection and disease, although only IgA antibodies correlated with reduction of shedding duration.
  (Starts with nice review of data on local (nasal) versus systemic antibody for other viruses.)
  Also discusses idea that "generalized" nasal immunity (total IgA) confers some short-term protection, could also be innate?
  In any case, points to evidence that resistance to colds is enhanced for a few weeks after infection.
  Interestingly, IgA concentrations (total) inversely correlated with neutralizing titers.
  Neutralizing serum antibody was strongly correlated with reduced disease and infection.
  Binding by IgG in serum or nasal IgA was also correlated with reduced disease / infection, but less than neutralizing serum antibody. Serum IgA and IgM less strongly correlated still.
  However, most were still infected judged by virus shedding, and serum neutralizing antibody only weakly associated with reduced shedding--however, IgA was correlated with reduced shedding.
  Those with a cold in prior 6 months were more resistant to infection and disease.

- [Schmidt1986, Rises in titers of antibody to human coronaviruses OC43 and 229E in Seattle families during 1975-1979.](https://www.ncbi.nlm.nih.gov/pubmed/3008551):
  Used ELISA to look at titers to OC43 and 229E in sera from 10 Seattle families (each with at least one pre-school child) every four months from 1975 to 1979.
  Only two children lacked antibodies to 229E, and one to OC43.
  Mean titers to both viruses increased with age.
  The average person exhibited 0.7 antibody rises per year across both virsues, with the frequencies of these rises being higher in younger children.
  With OC43, most of the rises occurred in the period from Oct/Nov to March/April.
  With 229E, they occurred across entire year more evenly.
  There were concurrent antibody rises to both OC43 and 229E in the same 4-month period three times (all in children); this is not different than the expected of five if infections occurred independently.
  So overall, this suggests no strong interference between OC43 and 229E.
  In some cases, repeated rises to same virus.
  At least 40% of antibody rises occurred in people who did not report any illness during that time period.

- [Callow1990, The time course of the immune response to experimental coronavirus infection of man](https://www.ncbi.nlm.nih.gov/pubmed/2170159):
  Human volunteers experimentally infected with 100 TCID50 of 229E.
  Important note is volunteers probably already had some immunity.
  IgG and IgA peaked at 3 weeks post-infection, were still elevated at 12 weeks, but were back to almost baseline at 52 weeks.
  10 of 15 volunteers were infected, 8 of 10 had colds from infection.
  The ones that did not get infected had higher nasal IgA and serum IgG pre-infection, and even uninfected had a transient boost in nasal IgA.
  The infected individuals had substantial increase in IgA and IgG at about 8 days post-infection.
  After a year, titers had declined in infected but still remained higher than pre-infection.
  On reinfection, the 5 that hadn't been infected before became infected a year later, whereas 6 of 9 of the previously infected also became re-infected---but only 1 of the former group had a cold.
  Virus shedding was also of much shorter duration on re-challenge.

- [Kunkel1993](https://reader.elsevier.com/reader/sd/pii/S0042682283713607):
  Performs first sequencing of OC43 CoV Spike, using the ATCC virus.
  Shows that the S protein has the hemagglutinating activity as well.
  States that S on virions is uncleaved, but can be cleaved to S1 / S2 with trypsin.
  This apparently differs from bovine coronavirus (BCV), which is already cleaved.
  They say that 9-O-acetylated sialic acide (Neu5,9Ac2) is receptor.
  It appears that OC43 prefers alpha-2,6 whereas BCV prefers alpha-2,3.

- [vanderHoek2004, Identification of a new human coronavirus](https://www.nature.com/articles/nm1024):
  Initial isolation of NL63 from a sick child.
  Virus grew on some monkey cells, but not Vero and some other cells.
  Then identified 7 more individuals with virus, and notes that some had point mutations.
  They then sequenced, and note that it has lowest GC of any CoV (34%).
  It's most similar to 229E, but has a 537 nt (179 aa) insertion relative to 229E.

- [Woo2005, Characterization and complete genome sequence of a novel coronavirus, coronavirus HKU1, from patients with pneumonia](https://www.ncbi.nlm.nih.gov/pubmed/15613317):
  Initial isolation of HKU1 from an elderly patient with pneumonia.
  Only has 32% GC.
  Figure 4 highlights residues important for receptor binding in MHV, most of which are conserved in HKU1 and OC43.

- [Vijgen2005, Complete Genomic Sequence of Human Coronavirus OC43: Molecular Clock Analysis Suggests a Relatively Recent Zoonotic Coronavirus Transmission Event](https://jvi.asm.org/content/79/3/1595):
  Reports complete genome of ATCC OC43 virus, which was isolated in 1967 and then lab passaged.
  It's most similar to BCoV.
  They build a date-stamped tree of OC43 and BCoV (with dates coming from BCoV) of Spike, and use this to estimate a divergence date of around 1890, although branches in BCoV lineage used to estimate rate are quite short compared to long branch to OC43.
  They also discuss OC43 passage history as: 7X human organ culture, 15X mouse brain, unknown in HRT-18 / Vero.
  They argue the transmission was BCoV (cows) -> OC43 (humans) rather than the other way around because OC43 has a deletion of 290 nt (not in Spike) that is not present in OC43 and other CoV.
  Also, the canine coronavirus CRCoV is quite close to OC43 in Spike, almost as much as BCoV.

- [Vijgen2005b, Circulation of genetically distinct contemporary human coronavirus OC43 strains](https://www.sciencedirect.com/science/article/pii/S0042682205002163):
  Sequence full genomes from two OC43-like CoV from 2003 / 2004, plus five additional Spikes.
  They are 99% identical to ATCC OC43 over whole gene, and about 96-97% amino-acid identity in Spike.
  The two strains they isolate one year apart are only 97% identical in Spike.
  Most substitutions in Spike are in S1.
  Has a tree of all the Spikes.
  All of the strains they seuqence lack a mutation in the S1-S2 cleavage site that is in the ATCC virus.
  They also note that the "Paris isolate" from St. Jean et al (2004) looks like an ATCC strain.

- [Vijgen2006](https://jvi.asm.org/content/jvi/80/14/7270):
  Sequencing PHEV and looked at similarity to OC43 and BCoV.
  PHEV, OC43, and BCoV are all closely related, and they estimate divergence in last few 100 years.
  But PHEV much more different than OC43 in Spike than rest of genome, more so than BCoV versus OC43.

- [Chibo2006, Analysis of human coronavirus 229E spike and nucleoprotein genes demonstrates genetic drift between chronologically distinct strains](https://www.microbiologyresearch.org/content/journal/jgv/10.1099/vir.0.81662-0):
  Analyzed Spike sequences from 229E isolates from between 1979 and 2004.
  Two deletions each of 3 amino acids were observed, as well as 67 codon differences from ATCC strain.
  Two N-linked glycans were gained.
  Seven amino acid substitutions were in RBD.
  Some of the mutations that appeared then fixed in all later isolates, overall a ladder-like tree.
  Identified some sites in Spike of positive selection, although not clear how well done that analysis was.

- [Djikman2008, Human Coronavirus NL63 and 229E Seroconversion in Children](https://jcm.asm.org/content/46/7/2368):
  Developed ELISA against N protein of NL63 and 229E, and looked in young children.
  All children had high levels of anti-N antibodies against both at birth presumably due to maternal antibodies.
  These decreased in first few months.
  Less than half of children had antibodies from new exposure at 18 months, but by 2.5-3.5 years more than half had antibodies, and by 6-10 years 80-90% had antibodies.

- [Haan2008, Cleavage of Group 1 Coronavirus Spike Proteins: How Furin Cleavage Is Traded Off against Heparan Sulfate Binding upon
Cell Culture Adaptation](https://jvi.asm.org/content/82/12/6078):
  Looks at S1 / S2 cleavage site.
  Mostly about feline coronavirus, but mentions how ATCC OC43 but not unpassaged OC43 has lost the RRSRR cleavage site.
  Also discusses some connection to heparan sulfate binding.

- [Lau2011, Molecular Epidemiology of Human Coronavirus OC43 Reveals Evolution of Different Genotypes over Time and Recent Emergence of a Novel Genotype due to
Natural Recombination](https://jvi.asm.org/content/jvi/85/21/11325.full.pdf):
  Sequenced 29 human OC43 from 2004 to 2011.
  Phylogeny is incongruent among genes, as expected due to recombination.
  The Hong Kong sequences here cluster in two groups with Belgium sequences in each group, indicating not strongly geographical structure.
  Estimates dN/dS for Spike of 0.27.
  Estimates common ancestor of all circulating Spikes was 1984, and if ATCC is included it's 1957.

- [Shirato2012, Differences in neutralizing antigenicity between laboratory and clinical isolates of HCoV-229E isolated in Japan in 2004–2008 depend on the S1 region sequence of the spike protein](https://www.microbiologyresearch.org/content/journal/jgv/10.1099/vir.0.043117-0):
  Compared neutralization of ATCC 229E and human isolates from Japan in 2004 or 2008.
  The ATCC and their isolates had many changes, mostly in S1 and especially upstream of the RBD (in N-terminal domain). The amino-acid identity was 89% for N-terminal domain, and 97% in the rest.
  They show a phylogenetic tree with two clades, but might actually be ladder-like--it's unclear how they rooted it.
  The used rabbit sera against the ATCC or one of the 2004 viruses.
  The ATCC anti-sera neutralized the ATCC isolate to 1:200, but not the 2004 isolate.
  The 2004-isolate anti-sera neutralized that virus to 1:80, but the ATCC virus by only 1:20.
  They then pseudotyped Spike on VSV, and confirmed differences in neutralization were due to Spike.
  They also made S1 / S2 chimeras, and confirmed that the differences were due to S1.
  They then collected human sera from adults and children, and show children mostly neutralize newer virus while adults neutralize ATCC virus comparably to newer virus (data here are not great, but seem to support).

- [Djikman2012, The dominance of human coronavirus OC43 and NL63 infections in infants](https://www.ncbi.nlm.nih.gov/pubmed/22188723):
  Followed young children for seroconversion to OC43, HKU1, NL63, and 229E.
  Children had HIV+ moms, but they say this doesn't matter.
  All children had high levels of antibodies to all CoV at birth, which they suggest maternal---these waned in first few months.
  OC43 and NL63 seroconversions were most common, and had happened in majority of patients by 20 months.
  229E and HKU1 were less common--not clear if this is universal or just years they chose.
  They suggest OC43 prevents HKU1 and NL63 prevents 229E infection, but no statistics to back this up are done.

- [Zhou2013, First infection by all four non-severe acute respiratory syndrome human coronaviruses takes place during childhood](https://bmcinfectdis.biomedcentral.com/articles/10.1186/1471-2334-13-433):
  Used serology against OC43, HKU1, 229E, and NL63 Spike in asymptomatic individuals from Beijing.
  Only 29 of 763 samples were negative for any CoV: >=70% were also positive for each individual virus.
  Seropositivity increased with age, although most children were seropositive for all by age 1-3.
  It's not clear to me how the assay specificity was established or validated.

- [Wertheim2013, A Case for the Ancient Origin of Coronaviruses](https://jvi.asm.org/content/87/12/7039.short):
  Argues that incorrect phylogenetic substitution models have led some papers to conclude common ancestor of all CoV just 10,000-ish years ago.
  Argues it is more likely to be 10s to 100s of millions of years ago.

- [Zhang2015, Genotype shift in human coronavirus OC43 and emergence of a novel genotype by natural recombination](https://www.sciencedirect.com/science/article/pii/S0163445314003764):
  Sequenced Spike from 65 OC43 infections between 2005 and 2012 in China.
  Recapitulated the four groups from Lau et al for Spike, but incongruities for N and RdRp.
  Again, clustering of Spike is not by geography.
  They suggest some recombination across genome, although not necessarily in Spike.
  They also plot yearly distribution of the different "groups" and suggest overall infections peak every other year and distributions change, although trends not super clear.

- [Ren2015, Genetic drift of human coronavirus OC43 spike gene during adaptive evolution](https://www.nature.com/articles/srep11451):
  Analyzes evolution of OC43.
  Assembles a large set of sequences, estimates evolutionary rates of about 5-9e-4 substitutions per year for all major genotypes.
  The dN/dS was about 0.2 - 0.3, and a few sites appeared to be under positive selection although this didn't appear to stand up to statistical testing of significance in most cases.
  Most of the apparently selected sites were in NTD, which binds receptor for OC43.
  Speculates about different evolutionary rates in different OC43 genotypes, but this doesn't seem well supported.

- [Kin2015, Genomic Analysis of 15 Human Coronaviruses OC43 (HCoV-OC43s) Circulating in France from 2001 to 2013 Reveals a High Intra-Specific Diversity with New Recombinant Genotypes](https://www.mdpi.com/1999-4915/7/5/2358):
  Sequenced 15 OC43 in France from clinical samples, and compared to other seuqences.
  In their trees, there are two major clades with the ATCC viruses inernal rather than ancestral.
  This gives a tMRCA for current gentoypes of 1932.
  They define a new cluster E for Spike, and has a deletion of 12 nt in S1 relative to other OC43 that is also present in BCoV, and is in the lectin domain of Spike.
  Therefore, it appears insertion happened in OC43 relative to BCoV.

- [Kin2016, Comparative molecular epidemiology of two closely related coronaviruses, bovine coronavirus (BCoV) and human coronavirus OC43 (HCoV-OC43), reveals a different evolutionary pattern](https://reader.elsevier.com/reader/sd/pii/S1567134816300776):
  Sequences some bovine CoV (BCoV) and performs phylogenetic analyses with OC43 and dromedary camel CoV that is closely related.
  Overall, they cluster into three clades, and they report that there is less recombination in BCoV than in OC43.
  Could be useful for looking at origin of OC43.

- [Morfopoulou2016, Human Coronavirus OC43 Associated with Fatal Encephalitis](https://www.nejm.org/doi/full/10.1056/NEJMc1509458):
  Describes an immunocompromised child who developed encephalitis that ultimately was shown to be due to OC43 in the brain.
  The virus was deep sequenced (not sure if data are available), and had lots of mutations in Spike that are cataloged in the supplementary material.

- [AlKhannaq2016](https://virologyj.biomedcentral.com/articles/10.1186/s12985-016-0488-4):
  Screened adults with respiratory symptoms in Malaysia for OC43 and HKU1.
  2.4% were positive for one for one of CoV, with OC43 and HKU1 having about equal prevalence.
  Several genes including Spike were sequenced.
  Builds trees of Spike.
  For OC43, the viruses are scattered throughout clades, and they estimate over tMRCA for OC43 to be 1951.
  Estimates a similar tMRCA for HKU1.
  None of the trees look ladderlike, although is the OC43 affected by mis-annotated ATCC sequences?
  Also appears to suggest recombination in Spike for some of the OC43.

- [Bakkers2017, Betacoronavirus Adaptation to Humans Involved Progressive Loss of Hemagglutinin-Esterase Lectin Activity](https://www.sciencedirect.com/science/article/pii/S1931312817300707):
  Shows OC43 has lost HE binding activity, and suggests that HKU1 has done the same.
  Importantly, this does not appear to be artifact of ATCC lab adapated OC43, but all OC43: indeed, current circulating ones have even less HE binding than ATCC lab adapted.

- [Oong2017, Identification and evolutionary dynamics of two novel human coronavirus OC43 genotypes associated with acute respiratory infections: phylogenetic, spatiotemporal and transmission network analyses](https://www.tandfonline.com/doi/full/10.1038/emi.2016.132):
  Sequenced more OC43 viruses in Malaysia.
  These are full-length versions of those in AlKhannaq2016.
  Identify some strains with an insertion of glycine at position 119 in Spike.
  Estimate evolutionary rate of 6-7e-4 nt per year in Spike.
  Shows some trees, and otherwise seems to do a lot of classification of genotypes into different groups and notes there is recombination among them.

- [Liu2017, Prevalence and genetic diversity analysis of human coronaviruses among cross-border children](https://link.springer.com/article/10.1186/s12985-017-0896-0):
  Collected nasal swabs from 3298 children (<6) crossing between Southern China and Hong Kong that had respiratory symptoms.
  About 1% had OC43 or HKU1, and a few had NL63 or 229E.
  Infers tree based on partial S sequences.
  The OC43 and HKU identified both fall into different clades, indicating multiple co-circulating genotypes.

- [Gorse2017, Interpatient mutational spectrum of human coronavirus‐OC43 revealed by illumina sequencing](https://onlinelibrary.wiley.com/doi/full/10.1002/jmv.24780):
  Deep sequenced OC43 from four acute infections.
  There is some amount of variation at some sites in Spike.
  Not much more can be concluded from the study here, although data are on SRA.

- [Wong2017, Receptor-binding loops in alphacoronavirus adaptation and evolution](https://www.nature.com/articles/s41467-017-01706-x):
  Crystalizes 229E RBD in complex with receptor, aminopeptidase N.
  The receptor binding is mediated by three loops, which have high sequence variation.
  First defined the minimal RBD region necessary for binding to receptor.
  They then crystalize, showing role of the three loops in interface, and that binding surface is different than in the porcine respiratory alphacoronavirus (PRCoV) that also binds the same protein receptor.
  Extreme variation in the amino acids in receptor binding loops: among known 229E sequences, 73% of sites in the receptor-binding residues or supporting residues vary among sequences, while only 11% of RBD surface residues outside the binding loops are variable.
  Supp Figure 6 shows an extremely ladder-like phylogenetic tree of the RBD sequences, which they divide into six classes in a way that seems fairly arbitrary.
  Shows that a monoclonal antibody neutralizing antibody binds to one class but not others.
  Also, receptor affinity appears to increase over time--but a caveat is that they are inserting class I sequence everywhere except binding loops, although most likely unproductive epistasis between binding loops and rest of RBD would lead to decreasing rather than increasing affinity.

- [Zhu2018, A novel human coronavirus OC43 genotype detected in mainland China.](https://www.tandfonline.com/doi/full/10.1038/s41426-018-0171-5):
  Sequences some OC43 viruses in China.
  Builds full-genome trees, which are slightly ladderlike and have sequences that were sequenced fall into two different clades on tree.

- [Zhang2018, Epidemiology characteristics of human coronaviruses in patients with respiratory infection symptoms and phylogenetic analysis of HCoV-OC43 during 2010-2015 in Guangzhou.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5788356/):
  Collected 13,000 throat swabs from adults and children with respiratory infections, and screened for CoV.
  Detected human CoV in 2.25% of caess, with the most being OC43, followed by 229E, NL63, and HKU1.
  Prevalence was highest in winter, and some years (non-consecutive) had higher peaks.
  15 Spikes were sequenced.
  The tree indicates that they were from several clades, and tree is **not** very ladderlike.

- [Kiuyka2018, Human Coronavirus NL63 Molecular Epidemiology and Evolutionary Patterns in Rural Coastal Kenya](https://academic.oup.com/jid/article/217/11/1728/4948258):
  Isolated NL63 in Kenya, sequenced Spike from household and inpatient samples.
  Trees show they fall in two clades.
  Several isolates had mutations in ACE2 binding region.
  They report quite a few repeat infections, but are only able to sequence from second infection.
  They say second infections had higher viral load.
  They talk a lot about repeat infections, but I'm not confident about their analysis and it's very confusing, so I'd want to dig a lot more before reading too much into any of this.

- [Kamau2019, Genome Sequences of Human Coronavirus OC43 and NL63, Associated with Respiratory Infections in Kilifi, Kenya](https://mra.asm.org/content/8/46/e00730-19):
  Simply reports two OC43 and one NL63 sequence from Kenya.
  Only notable thing is that the two OC43 genotypes cluster on differnt parts of the OC43 tree, indicating not strong geographical structure.
  Neither the OC43 or NL63 full-genome trees look ladderlike, although no tests of recombination are reported.

- [Tortorici2019, Structural basis for human coronavirus attachment to sialic acid receptors](https://www.nature.com/articles/s41594-019-0233-y):
  Structure of OC43 Spike in complex with sialic acid (9-O-Ac-Sia).
  They only observe domain B ("RBD") in closed conformation, and say similar has been observed for NL63 and HKU1.
  Identify the sialic acid binding site on domain A (NTD), which is in a pocket sort of on the side.
  The Kd for binding was 50 uM with fast on and off, but they point out that there is probably lots of avidity.
  They validate the receptor-binding residues they find using pseudotyped VSV-G.
  They also show that sialic acid binding or pH do not trigger fusion.
  Say that binding in BCoV is similar to OC43.

- [Szczepanski2019, Canine Respiratory Coronavirus, Bovine Coronavirus, and Human Coronavirus OC43: Receptors and Attachment Factors](https://www.mdpi.com/1999-4915/11/4/328):
  Compares receptors among OC43, canine corinavirus, and BCoV.
  They show all three agglutinate mouse red blood cells, and that sialidiase treatment reduces viral attachment to cells by all three viruses by 50-ish %.
  However, sialidase treatment does not reduce infection of cells (possibly because removal is incomplete?).
  They also do re-attachment, and overall show that canine coronavirus prefers 2,3-sialic acid while BCoV and OC43 prefer 2,6-sialic acid.
  They next look at idea that HLA-I is receptor, and show that blocking it blocks infection by canine CoV and BCoV but not OC43.
  They show that heparin sulfate partially blocks attachment by all three viruses, especially OC43 and BCoV.
  They then looked at clinical strains, and found no effect of heparin sulfte or HLA-I blockage for clinical OC43, but an effect of soluble sialic acid on reducing infection.
  They also say other literature says affinity for heparin sulfate arises primarily as a cell culture adaptation.

- [Maurier2019](https://www.sciencedirect.com/science/article/pii/S0042682219300728):
  Describes protocol to sequence full genome of OC43 by making tiled PCR amplicons, and sequence 11 isolates.

- [Hulswit2019, Human coronaviruses OC43 and HKU1 bind to 9-O-acetylated sialic acids via a conserved receptor-binding site in spike protein domain A](https://www.pnas.org/content/116/7/2681):
  Use crystallography and biochemical assays to show that OC43 and HKU1 both bind to sialic acid receptor using the same site (they call it "site B") on S1 domain A, which apparently is a different part of domain A than some other papers had argued.

- [Li2019, The human coronavirus HCoV-229E S-protein structure and receptor binding](https://elifesciences.org/articles/51230):
  Crystallizes RBD-receptor interaction of 229E viruses from classes III to V (these are more recent viruses).
  All bind receptor in a very similar way.
  Talks about how these non-lab-adapted RBDs have some differences from early lab adapted 229E.
  Also solve a cryo-EM structure of full Spike.
  In all their cryo-EM, RBD is in down conformation, but they speculate it can go up.
  Also, speculates how receptor-binding that puts RBD in up conformation might promote S2 fusion.

- [Nilsson2020, Fatal encephalitis associated with coronavirus OC43 in an immunocompromised child](https://www.tandfonline.com/doi/full/10.1080/23744235.2020.1729403):
  Describes a severely immunocompromised child who had respiratory OC43 and then was diagnosed with encephalitis.
  Ultimately a brain biopsy found OC43, and the child was also positive for respiratory OC43 for many tests before dying.

- [Beury2020, Use of whole-genome sequencing in the molecular investigation of care- T associated HCoV-OC43 infections in a hematopoietic stem cell transplant
unit](https://www.sciencedirect.com/science/article/pii/S1386653219302367):
  Deep sequenced some OC43 viruses in stem-cell transplant patients in France.
  Nothing seems remarkable about these sequences, they cluster with other OC43 sequences.

- [Nickbakhsh2020, Epidemiology of Seasonal Coronaviruses: Establishing the Context for the Emergence of Coronavirus Disease 2019](https://academic.oup.com/jid/advance-article/doi/10.1093/infdis/jiaa185/5820656):
  Looks at epidemiology of seasonal coronaviruses in UK.
  Lots of data on seasonality, age distribution, etc.
  Major findings include all are seasonal, and there may be some cross immunity between NL63 and 229E based on alernating large seasons.

- [Huang2020, A systematic review of antibody mediated immunity to coronaviruses: antibody kinetics, correlates of protection, and association of antibody responses with severity of disease](https://www.medrxiv.org/content/10.1101/2020.04.14.20065771v1):
  Very nice review on coronavirus immunity including endemic and zoonotic ones.
  Includes evidence from challenge studies on effects of immunity, duration of antibodies, and seroprevalence.

- [Monto2020, Coronavirus Occurrence and Transmission Over 8 Years in the HIVE Cohort of Households in Michigan](https://academic.oup.com/jid/advance-article/doi/10.1093/infdis/jiaa161/5815743):
  Looks at seasonal CoV in Michigan over 8 years (2010 to 2018).
  All households had children.
  The fraction of acute respiratory illnesses associated with CoV ranged from 8 to 16%.
  All were sharply seasonal, with the peak in January / February.
  The incidence for each individual CoV tended to peak in alternate years (or sometimes only every 2 or 3 years), with 229E and NL63 out of sync, and also HKU1 and OC43 perhas out of sync.
  Except for 229E, all CoV had the highest incidence in children <5 years, but flat with age after that.
  The secondary attack rate in households was 7-13%, with an average of 3-4 days before onset of secondary cases.

- [Gorse2020, Antibodies to coronaviruses are higher in older compared with younger adults and binding antibodies are more sensitive than neutralizing antibodies in identifying coronavirus‐associated illnesses](https://www.ncbi.nlm.nih.gov/pubmed/32073157):
  Looks at adults infected with seasonal CoV.
  Both binding and neutralizing antibodies were higher in adults.
  Only 36% of infections with rise in binding antibodies also had rise in neutralizing antibodies.

- [Kissler2020, Projecting the transmission dynamics of SARS-CoV-2 through the postpandemic period](https://science.sciencemag.org/content/early/2020/05/11/science.abb5793):
  Primarily models possible scenarios for SARS-CoV-2, but contains some analysis of R as a function of season and cross immunity for OC43 and HKU1.

- [Kellam2020, The dynamics of humoral immune responses following SARS-CoV-2 infection and the potential for reinfection](https://www.microbiologyresearch.org/content/journal/jgv/10.1099/jgv.0.001439):
  Nice review about antibodies, immune durability, and re-infection with CoV, both SARS-CoV / MERS-CoV, human endemic, and SARS-CoV-2.
  Reviews evidence that antibodies might wane over time, disease severity affects antibody titers (more disease = higher titers), re-infection with seasonal CoV, and cross-immunity of HKU1 / OC43 and 229E / NL63, but not between those beta- and alpha- groups.

- [Edridge2020, Human coronavirus reinfection dynamics: lessons for SARS-CoV-2](https://www.medrxiv.org/content/10.1101/2020.05.11.20086439v1):
  Uses titer rises to look for infections over many years in a cohort with seasonal coronaviruses.
  There are lots of reinfections, with median time to reinfection ranging from 30 to 48 months for the different coronaviruses.
  There does not appear to be any correlation between alpha- and beta-coronavirus infections.
  The infections happen less in summer, more in winter.
  They report some infected individuals have antibodies that react with SARS-CoV-2 N.

- [Lang2020, Coronavirus hemagglutinin-esterase and spike proteins co-evolve for functional balance and optimal virion avidity](https://www.biorxiv.org/content/10.1101/2020.04.03.003699v1.full):
  Basically, they are looking at the bovine coronavirus predecessor of OC43, which has a HE protein in addition to Spike. The HE can bind but appears to primarily cleave receptor, sort of like flu NA. The receptor is some sort of sialic-acid like thing. When they knock out the HE binding ability (which presumably reduces its overall cleavage by making it less avid for its substrates), they get compensatory mutations in Spike that reduce its binding. The use this to argue for a balance between cleaving and binding similar to HA.
  They also have some results about how their lab passaging appears to generate “cooperating” swarms of high / low binding viruses that seem reminiscent of @anabundance’s cooperating flu viruses, and they cite that paper, although it’s hard for me to tell without digging a lot more into their data how robust the conclusion of cooperation actually is.
  I also cannot quite tell if this is relevant to OC43 itself or just the lab-adapted OC43 ATCC type strain. They have some discussion about how the ATCC strain is lab adapted, but never elaborate on how this relates to circulating viruses.
  Another thing that might be of relevance to @13reguia @Laurel K for OC43 pseudotyping is that they show that for the bovine coronavirus you have to have active HE protein (either on virion or soluble) to mediate good infection, similar to how you need NA to release HA-expressing virions. So maybe this will be a consideration for producing OC43 Spike pseudotyped lentiviruses. Because I can’t tell if the RDE we use for flu would also cut the 9-O-Ac-Sias that OC43 binds (help @Karen!), I’m not sure if this is relevant to us?
  Finally, they seem to keep referring to the HE cleavage activity as important for promoting infection by allowing release of incoming virions from “unproductive” receptors so they can bind productive ones. I’m not sure why they don’t also think it could simply affect virion release as people usually envision for flu NA.

- [Gilca2020, Common human coronaviruses seem at least as severe as influenza in patients hospitalized with acute respiratory infection: results from 8-year hospital-based surveillance in Quebec, Canada](https://academic.oup.com/jid/advance-article/doi/10.1093/infdis/jiaa477/5882014): compares frequency of human endemic coronaviruses and influenza in individuals hospitalized with acute respiratory illness. Influenza is more common in these individuals for all age gorups except children, with influenz becoming more common with age, and HCoV becoming less common with age (Fig 1). Note that this is among individuals hospitalized with ARI, so by definition they have something, so this is more relative within age groups than absolute across age groups.

- [Aldridge2020, Seasonality and immunity to laboratory-confirmed seasonal coronaviruses (HCoV-NL63, HCoV-OC43, and HCoV-229E): results from the Flu Watch cohort study](https://wellcomeopenresearch.org/articles/5-52): Uses viral testing to look for NL63, OC43, and 229E infections in the winters from 2006 to 2009. They swabbed individuals with illnesses and tested for CoV. Rates of infection were highest in winter. Overall detected 0.4 infections per 100 person weeks for any CoV, with some re-infections with a CoV but never of same type--this is statistically significant evidence against reinfection with same type. Notably, compared to other studies this one uses PCR testing so should be more stringent, but probably less sensitive as individual has to have a cold to be tested.

- [Kistler2020, Evidence for adaptive evolution in the receptor-binding domain of seasonal coronaviruses](https://www.biorxiv.org/content/10.1101/2020.10.30.352914v1): Uses a variety of tests to show that S1 domain of 229E and OC43 evolve under positive selection. Also some nice trees showing 229E has one major lineage, but OC43 has two.
